docs/api-docs/provider-apis/tax/overview.mdx

```diff
@@ -91,7 +77,6 @@ Make sure to also review our [app development best practices](/docs/integrations
   If you have registered your app in the Developer Portal but have not submitted it for approval because it's still in development, the app will be in a Draft state. This means your app can only be installed on stores owned by the same email address as the Developer Portal account email.
 </Callout>
 
-
 ## Installing the app
```

Comment:
👁️ I've adjusted the whitespace to remove any double blank lines - they didn't appear to serve a purpose.

Reply:
👍